spopt/tests/test_c_p_median.py

```diff
@@ -23,8 +28,10 @@ def setup_method(self) -> None:
         lattice = spaghetti.regular_lattice((0, 0, 10, 10), 9, exterior=True)
         ntw = spaghetti.Network(in_data=lattice)
         gdf = spaghetti.element_as_gdf(ntw, arcs=True)
+        net_buffer = gdf["geometry"].buffer(0.2)
+        net_space = net_buffer.union_all() if GPD_GE_100 else net_buffer.unary_union
         street = geopandas.GeoDataFrame(
-            geopandas.GeoSeries(gdf["geometry"].buffer(0.2).unary_union),
+            geopandas.GeoSeries(net_space),
```

Comment:
A side note - since we are reusing this across so many places, this setup method should've been a global fixture.

Reply:
LOL. Great minds think alike. Already planning for implementing that in a subsequent PR.